src/React.AspNet/project.json

```diff
@@ -1,43 +1,52 @@
 {
-    "version": "2.3.0-*",
-    "configurations": {
-        "Debug": {
-            "compilationOptions": {
-                "define": [ "DEBUG", "TRACE", "ASPNET5" ]
-            }
-        },
-        "Release": {
-            "compilationOptions": {
-                "define": [ "RELEASE", "TRACE", "ASPNET5" ],
-                "optimize": true,
-                "warningsAsErrors": true
-            }
-        }
+  "version": "2.3.0-*",
+  "authors": [ "Daniel Lo Nigro" ],
+  "packOptions": {
+    "owners": [ "Daniel Lo Nigro" ],
+    "licenseUrl": "https://github.com/reactjs/React.NET#licence",
+    "iconUrl": "http://facebook.github.io/react/img/logo_og.png",
+    "copyright": "Copyright 2014-Present Facebook, Inc",
+    "title": "ReactJS.NET (MVC Core)",
+    "description": "ReactJS and Babel tools for ASP.NET Core, including ASP.NET MVC Core. Please refer to project site (http://reactjs.net/) for full installation instructions, usage examples and sample code",
+    "tags": [ "asp.net", "mvc", "asp", "javascript", "js", "react", "facebook", "reactjs", "vnext" ],
+    "projectUrl": "http://reactjs.net/"
+  },
+  "configurations": {
+    "Debug": {
+      "buildOptions": {
+        "define": [ "DEBUG", "TRACE", "ASPNET5" ]
```

Comment:
I wonder if we should rename the `ASPNET5` define too. It's referenced in a few source files via `#ifdef ASPNET5`